apps/govern/components/VeOlas/useVeolasComponents.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  formatWeiNumber,
+  getCommaSeparatedNumber,
+  getFormattedDate,
+  getFullFormattedDate,
+} from 'common-util/functions';
+import { useFetchBalances } from 'hooks/useFetchBalances';
+
+import { InfoCard } from './InfoCard';
+
+const getTotalVotesPercentage = (
+  votingPower: string | undefined,
+  totalSupply: string | undefined,
+) => {
+  if (votingPower && totalSupply) {
+    const votingPowerInPercentage = ((Number(votingPower) / Number(totalSupply)) * 100).toFixed(2);
+    return formatWeiNumber({ value: votingPowerInPercentage });
+  }
+
+  return null;
+};
+
+/**
+ * This hook is used to get the components
+ */
+export const useVeolasComponents = () => {
```

Comment:
feels like a strange pattern; 
- we can call fetch balances in the parent component that uses these components
- these components can be imported as/when needed using normal imports?

Reply:
fixed